UKHomeOffice/sgmr-service
# SPCR-407 Remove useContext from account pages

### AC / Description of changes
- remove useContext for user
- refactor account pages to new structure
- update ActionEntity to handle account deletion


### To Test
- go to /account
- > page should load with your details and an edit button
- go to another page
- enter `/account/edit` in the url bar
- > page should load the edit account form with your details prefilled
- change details
- click exit without saving
- > you should be returned to the account page without the details changing
- click edit
- > page should load the edit account for with your details prefilled
- change details
- click save changes
- > you should be returned to the account page with the changed details showing
- click edit
- click delete your account
- > you should see an are you sure page
- click no
- > you should be returned to the account page
- click edit
- click delete your account
- click yes to are you sure
- > you should be taken to a confirmation of account deleted page
- check sessionStorage
- > you should no longer have a token
- try to log in
- > you should not be able to log in


### Notes
This will delete your account, you can create another one through Postman or via the website and it will send you the confirmation email

---
* remember to merge to feature branches not master *
